ui_testing/testcases/basic_tests/test04_testunits.py

```diff
@@ -777,169 +777,71 @@ def test023_duplicate_test_case(self):
         self.info('assert there is a new test unit')
         self.assertGreater(new_test_units, old_test_units)
 
-    @parameterized.expand(['unitsub', 'unitsuper'])
-    def test029_qualitative_test_unit_with_sub_and_super_scripts_appears_in_exported_sheet(self,
-                                                                                           unit_with_sub_or_super):
+    @parameterized.expand([('unitsub', 'qualitative'),
+                           ('unitsub', 'quantitative'),
+                           ('unitsuper', 'qualitative'),
+                           ('unitsuper', 'quantitative')])
+    def test023_test_unit_with_sub_and_super_scripts_appears_in_exported_sheet(self, unit_with_sub_or_super, type):
```

Comment:
Run those test cases in travis

Reply:
This test case isn't related to this branch, so I don't need to run them in travis